app/pages/lab/workflow.cjsx

```diff
@@ -53,6 +54,11 @@ EditWorkflowPage = React.createClass
     deletionError: null
     workflowCreationInProgress: false
 
+  componentWillReceiveProps: (nextProps) ->
+    if nextProps.workflow isnt @props.workflow
+      ReactDOM.findDOMNode(@refs.defaultWorkflow).defaultChecked = nextProps.project.configuration.default_workflow is nextProps.workflow.id
```

Comment:
I added this in to fix the bug I described. When switching between workflow editors, if the first workflow had these options enabled, when switching to the another workflow that do not have them enabled, the inputs would stay checked giving the false impression that these options were enabled for the second workflow.